docs/reference/commandline/build.md

```diff
@@ -63,11 +63,11 @@ Options:
 
 ## Description
 
-Builds Docker images from a Dockerfile and a "context". A build's context is
-the files located in the specified `PATH` or `URL`. The build process can refer
-to any of the files in the context. For example, your build can use an
-[*ADD*](../builder.md#add) instruction to reference a file in the
-context.
+The `docker build` command builds Docker images from from a Dockerfile and a
```

Comment:
ahah  @thaJeztah after *is is*, *from from* 😂 I think you're doing it to make sure we review it correctly 😝